pkg/v1/providers/config_default.yaml

```diff
@@ -596,15 +596,36 @@ AVI_CONTROL_PLANE_HA_PROVIDER: false
 #! Antrea CNI configuration
 #! ---------------------------------------------------------------------
 ANTREA_NO_SNAT: false
+ANTREA_DISABLE_UDP_TUNNEL_OFFLOAD: false
 ANTREA_TRAFFIC_ENCAP_MODE: "encap"
+ANTREA_EGRESS_EXCEPT_CIDRS: ""
+ANTREA_NODEPORTLOCAL_ENABLED: true
+ANTREA_NODEPORTLOCAL_PORTRANGE: 61000-62000
+ANTREA_PROXY_ALL: true
```

Comment:
I think the default value for ANTREA_PROXY_ALL is false.  This option will be enabled if you want to remove kube-proxy and use antrea-proxy to replace kube-proxy.